routes/workoutlog.js

```diff
@@ -0,0 +1,12 @@
+// For workoutlog data
+const express = require('express'); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const express = require("express"); 
```